examples/plot_text_classification.py

```diff
@@ -0,0 +1,169 @@
+"""
+scikit-learn model cards
+--------------------------------------
+
+This guide demonstrates how you can make the best of this package to host text
+classification models.
+"""
+
+# %% Imports
+# =======
+# First we will import everything required for the rest of this document.
+
+import pickle
+from pathlib import Path
+from tempfile import mkdtemp, mkstemp
+
+import pandas as pd
+import sklearn
+from sklearn.datasets import fetch_20newsgroups
+from sklearn.feature_extraction.text import CountVectorizer, TfidfTransformer
+from sklearn.metrics import (
+    ConfusionMatrixDisplay,
+    accuracy_score,
+    classification_report,
+    confusion_matrix,
+    f1_score,
+)
+from sklearn.model_selection import train_test_split
+from sklearn.naive_bayes import MultinomialNB
+from sklearn.pipeline import Pipeline
+
+from skops import card, hub_utils
+
+# %% Data
+# ====
+# We will use 20 newsgroups dataset from sklearn. We will define categories of
+# this dataset ourselves as targets are integers.
+
+categories = ["alt.atheism", "soc.religion.christian", "comp.graphics", "sci.med"]
+twenty_train = fetch_20newsgroups(
+    subset="train", categories=categories, shuffle=True, random_state=42
+)
+
+twenty_validation = fetch_20newsgroups(
+    subset="test", categories=categories, shuffle=True, random_state=42
+)
+
+X_train, X_test, y_train, y_test = train_test_split(
+    twenty_train.data, twenty_train.target, test_size=0.3, random_state=42
+)
+
+# %% Train a Model
+# =============
+# To train a model, we need to convert our data first to vectors. We will use
+# CountVectorizer and TFIDFVectorizer in our pipeline. We will fit a Multinomial
+# Naive Bayes model with the outputs of the vectorization.
+
+model = Pipeline(
+    [
+        ("vect", CountVectorizer()),
+        ("tfidf", TfidfTransformer()),
+        ("clf", MultinomialNB()),
+    ]
+)
+
+model.fit(X_train, y_train)
+
+# %% Inference
+# =============
+# Let's see if the model works.
+
+docs_new = ["Statistical machine learning is the best."]
+predicted = model.predict(docs_new)
+print(twenty_train.target_names[predicted[0]])
```

Comment:
if instead of `target` we use `target_names`, here we simply get the string value.

Reply:
same here